### ISSUE TITLE
Add TrackPoint extensions

### ISSUE THREAD

**Issue by fabulator** [pdhoopr/gps-to-gpx#6]
Hi,

I would like to use track point extensions in generated xml files so I can add heart rate, candence and other metrics. It should look like this:

```xml
<trkpt lat="54.8591470" lon="-1.5754310">
<ele>29.2</ele>
<time>2015-07-26T07:43:42Z</time>
<extensions>
<gpxtpx:TrackPointExtension>
<gpxtpx:atemp>11</gpxtpx:atemp>
<gpxtpx:hr>92</gpxtpx:hr>
<gpxtpx:cad>0</gpxtpx:cad>
</gpxtpx:TrackPointExtension>
</extensions>
</trkpt>
```

It is simple to implement:

```javascript
if (point.extensions) {
      const trackExtension = trkpt.ele('extensions').ele('gpxtpx:TrackPointExtension');
      Object.keys(point.extensions).forEach((key) => {
        trackExtension.ele(`gpxtpx:${key}`, point.extensions[key]);
      });
    }
```

Could you add it to this library?

**Comment by pdhoopr** [comment]
Hi there,

I've just released v1.2.0 which should let you add extensions as you've requested (hopefully 🙂). Let me know if something doesn't work as expected.

Thanks for the request!

**Comment by fabulator** [comment]
@pdhoopr I tried it and it works great! Thanks.
